awxkit/awxkit/cli/resource.py

```diff
@@ -140,7 +140,7 @@ def handle(self, client, parser):
         if fmt == 'json':
             data = json.load(client.stdin)
         elif fmt == 'yaml':
-            data = yaml.safe_load(client.stdin)
```

Comment:
There's some alarm bells going off for me for the security implications here:

https://pyyaml.org/wiki/PyYAMLDocumentation

> Warning: It is not safe to call yaml.load with any data received from an untrusted source! yaml.load is as powerful as pickle.load and so may call any Python function. Check the yaml.safe_load function though.

Reply:
Yes, I am aware of `load` vs `safe_load`, but this is not an issue here, since we do specify a "safe" loader.

The safety implications of `yaml.load()` stem from the fact that `yaml.Loader` is unsafe. Here, we use `awxkit.yaml_file.Loader` which inherits from `yaml.SafeLoader`, so to me it seems equivalent from a safety standpoint.

Also, `yaml.safe_load()` is simply an alias for `yaml.load(..., loader=SafeLoader)`.

Reply:
Ah, yep!

Sorry, I wasn't looking closely enough.  Thanks, @neoaggelos.